diagnostic.md

```diff
@@ -55,7 +55,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+absolute
```

Comment:
Yes, but it asks you to further explain your answer: the first / represents the root of the system.